docs/config.md

```diff
@@ -26,9 +26,9 @@ If you do not already have a [Landofile](https://docs.lando.dev/config/lando.htm
 
 Note that if the above config options are not enough, all Lando recipes can be further [extended and overriden](https://docs.lando.dev/config/recipes.html#extending-and-overriding-recipes).
 
-## Choosing a php version
+## Choosing PHP/database/Solr versions
```

Comment:
@reynoldsalec might be good to just call this `Choosing versions` or `Choosing services` or something.